cypress/integration/creatorOnboardingFlows/creatorSignup.spec.js

```diff
@@ -71,7 +71,7 @@ describe('Creator Signup Page', () => {
     cy.get('.js-eye').should('be.visible');
     cy.get('.js-eye-off').should('not.be.visible');
 
-    cy.get('.js-creator-password-visibility').click({ force: true });
+    cy.findByRole('button', { name: 'Toggle password visibility' }).click();
     cy.get('input[name="user[password]"]').should('have.attr', 'type', 'text');
```

Comment:
```suggestion
    cy.findByLabelText('Password').should('have.attr', 'type', 'text');
```

I haven't tried this out, but it should work and could help us test the accessibility at the same time (not blocking though, just a thought)